.travis.yml

```diff
@@ -1,14 +1,11 @@
 language: ruby
 rvm:
   - 1.9.3
-  - 2.0.0
-  - 2.1.0
-  - jruby
-  - rbx-2.1.1
-  - ruby-head
+  - 2.2.3
+  - jruby-1.7.4
+  - jruby-1.7.23
 matrix:
   allow_failures:
-    - rvm: ruby-head
-    - rvm: rbx-2.1.1
-notifications:
-  irc: "irc.freenode.org#adhearsion"
+    - rvm: jruby-1.7.4 # tends to dead-lock quite often
```

Comment:
:worried: Yeah, I had some funky results, sounds like you probably did as well.
My observations after checking out 3ce90c9 and running `bundle exec rake`:
* Using jruby-1.7.4
    * On an old (but still valid) Gemfile.lock: ...The test suite simply hung after a few dozen tests.  
    * After running `bundle update`:  ...Didn't get dead-lock, but instead "1839 examples, 13 failures, 31 pending"
* Using jruby-1.7.23
    * On an old (but still valid) Gemfile.lock: ...The test suite "1839 examples, 1115 failures, 26 pending"
    * After running `bundle update`:  The good news: "1839 examples, 0 failures, 31 pending"
* Sorry if too verbose, but [my full results are in a gist here](https://gist.github.com/sfgeorge/6204604d6df1b0cecce8)

So a few questions for you - and please push back if these are too difficult...
* I assume that the 13 failures that work on 1.7.23 but not on 1.7.4 would be quite difficult to just "fix", or to fix within the realm of Punchblock and not updating JRuby itself. Correct?
* Is it feasible to compare the 2 Gemfile.lock(s) in the gist link above and possibly identify which gems need to be upgraded in order to reduce punchblock test suite failures down to the lucky 13? If we can "tighten" punchblock.gemspec appropriately, then I'm assuming this will also _force_ apptastic to update those gems as well.  The hope here is of course to avoid production issues with outdated gems.

What are your thoughts on all of this?  Thank you Karol.